code/lib/cli/src/add.ts

```diff
@@ -33,6 +37,8 @@ const getVersionSpecifier = (addon: string) => {
   return groups ? [groups[1], groups[2]] : [addon, undefined];
 };
 
+const isSemVer = (version: string) => /^\d+\.\d+\.\d+(-.*)?$/.test(version);
```

Comment:
Can we use the semver library for this rather than writing our own function? I believe we already depend on it